cerberus/kubernetes/client.py

```diff
@@ -39,18 +39,24 @@ def list_pods(namespace):
 def monitor_nodes():
     nodes = list_nodes()
     notready_nodes = []
+    node_kerneldeadlock_status = None
     for node in nodes:
         try:
             node_info = cli.read_node_status(node, pretty=True)
         except ApiException as e:
             logging.error("Exception when calling \
                            CoreV1Api->read_node_status: %s\n" % e)
         for condition in node_info.status.conditions:
-            if condition.type == "Ready":
-                node_status = condition.status
+            if condition.type == "KernelDeadlock":
+                node_kerneldeadlock_status = condition.status
+            elif condition.type == "Ready":
+                node_ready_status = condition.status
             else:
                 continue
-        if node_status != "True":
+        if (
+            (node_kerneldeadlock_status != "False" or node_kerneldeadlock_status == None)        # noqa
```

Comment:
Enabling node problem detector is optional and is taken care by the user, we shouldn't be checking the node_kerneldeadlock_status when it's not enabled.

Reply:
i can set the default value of `node_kerneldeadlock_status` to False. This way if npd is not enabled it'll not throw any errors. Thoughts?